pkg/tests/api/loki/loki_test.go

```diff
@@ -75,6 +75,57 @@ func TestIntegrationLoki(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	t.Run("When calling /api/ds/query forward alerting-related headers", func(t *testing.T) {
```

Comment:
Nit. I would add this to the existing "When calling /api/ds/query should set expected headers on outgoing HTTP request" since integration tests are quite expensive to run.

Reply:
i see. was trying to separate the "set by loki-datasource" and "set by generic grafana code" situations, but can merge them, no problem.